lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -6105,7 +6105,7 @@ class PullbackEmitter final : public SILInstructionVisitor<PullbackEmitter> {
     // Allocate local buffer and initialize to zero.
     auto bufObjectType = getRemappedTangentType(originalBuffer->getType());
     auto *newBuf = localAllocBuilder.createAllocStack(
-        originalBuffer.getLoc(), bufObjectType);
+        RegularLocation::getAutoGeneratedLocation(), bufObjectType);
```

Comment:
Note: this code is constructing a temporary buffer inside `PullbackEmitter::getAdjointBuffer` - maybe it's even correct for this buffer to have a synthesized location (because it's an auxiliary variable that doesn't directly relate to user-written variables).